tests/consumers/duckdb_consumer.py

```diff
@@ -0,0 +1,64 @@
+import os
+import string
+from collections.abc import Iterable
+
+import pyarrow as pa
+
+
+class DuckDBConsumer:
+    """
+    Implementation of the DuckDB substrait consumer Class for testing
+    """
+
+    @staticmethod
+    def run_substrait_query(db_connection, sql_query: str) -> pa.Table:
+        """
+        Convert the SQL query into a substrait query plan and run the plan against DuckDB.
+
+        Parameters:
+            db_connection:
+                DuckDB connection.
+            sql_query:
+                A json formatted string representing the substrait query plan.
```

Comment:
What does json formatted string representing the substrait query plan means in this context? Isn't this just a plain SQL query string?

Reply:
Thanks for catching this, i'll update it.